Client/Clique/Modules/Settings/SecurityViewController.swift

```diff
@@ -0,0 +1,184 @@
+import UIKit
+import NVActivityIndicatorView
+import IQKeyboardManagerSwift
+
+class SecurityViewController: UIViewController {
+    
+    @IBOutlet private weak var closeButton: UIButton!
+    @IBOutlet private weak var oldPasswordTextField: UITextField!
+    @IBOutlet private weak var newPasswordTextField: UITextField!
+    @IBOutlet weak var repeatNewPasswordTextField: UITextField!
+    @IBOutlet private weak var passwordCheckLabel: UILabel!
+    @IBOutlet private weak var matchingPasswordsLabel: UILabel!
+    
+    var iconClick = false
+    let oldPasswordShowButton = UIButton(type: .custom)
+    let newPasswordShowButton = UIButton(type: .custom)
+    let repeatNewPasswordShowButton = UIButton(type: .custom)
+    
+    @IBOutlet weak var savePasswordButton: UIButton!
+    private let settingsService = SettingService()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        oldPasswordTextField.isSecureTextEntry = true
+        newPasswordTextField.isSecureTextEntry = true
+        repeatNewPasswordTextField.isSecureTextEntry = true
+
+        self.showButton()
+        
+        newPasswordTextField.addTarget(self, action: #selector(checkAndDisplayError(textfield:)), for: .editingChanged)
+        repeatNewPasswordTextField.addTarget(self, action: #selector(compareAndDisplay(textfield:)), for: .editingChanged)
+    }
+    
+    @IBAction func closeSettingsViewController(_ sender: UIButton) {
+        dismiss(animated: true, completion: nil)
+    }
+    
+    func getPasswordData() -> PasswordData? {
+        
+        guard
+            let oldPassword = oldPasswordTextField.text,
+            let newPassword = newPasswordTextField.text,
+                !oldPassword.isEmpty && !newPassword.isEmpty
+        else {
+            return nil
+        }
```

Comment:
```suggestion
        else { return nil }
```
Whenever you are using `return` or `return something` you can display it in one line